sub-packages/bionemo-geneformer/tests/bionemo/geneformer/scripts/test_train_geneformer.py

```diff
@@ -29,15 +29,21 @@
 from bionemo.testing import megatron_parallel_state_utils
 
 
-data_path: Path = load("single_cell/testdata-20240506") / "cellxgene_2023-12-15_small" / "processed_data"
+@pytest.fixture
+def data_path() -> Path:
+    """Gets the path to the directory with with cellx small dataset in Single Cell Memmap format.
+    Returns:
+        A Path object that is the directory with the specified test data.
+    """
+    return load("single_cell/testdata-memmap-format", source="pbss") / "cellxgene_2023-12-15_small_mmap"
```

Comment:
don't set source manually, use the environment variable instead. Note on how to do that as the last comment below.